SampleDanger/ViewController.swift

```diff
@@ -17,7 +17,635 @@ class ViewController: UIViewController {
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                         print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                        print("checkdfgh")
+                                         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: SampleDanger/ViewController.swift;
  Line: 648;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)